namespace.go

```diff
@@ -133,3 +133,7 @@ func (ns *Namespace) getHTTPSHostURI() string {
 func (ns *Namespace) getEntityAudience(entityPath string) string {
 	return ns.getAMQPHostURI() + entityPath
 }
+
+func (ns *Namespace) getEntityManagementPath(entityPath string) string {
```

Comment:
I [added getters](https://github.com/marstr/azure-service-bus-go/blob/1bd35f2be21e2a0bcd306513fc12663bde81d92b/queue.go#L292) to the `entity` type for this in my branch.

I can spin up a new PR to add those in quick so that you can take a dependency on them instead of adding this method.

Reply:
Closing the loop, I opened #60 for my side of this.

Reply:
Looking at the PR your getter returns the full path (including the amqp host URI). The code I've written is just after the entityName + "/$managment" (Sorry my bad on naming I started out using the full path before I realized it was incorrect then didn't fix up the method naming. 

What would be the best way to approach this?

Reply:
Done